### PR TITLE
[34132] Extend search autocompleter with useful information (improve the status badges)

### PR DESCRIPTION
**Pull request by bsatarnejad**
Making the status badges in global search autocompleter consistent with other places like boards.

### CODE REVIEW

File: frontend/src/app/modules/global_search/input/global-search-input.component.ts [hunk 1]
```diff
@@ -425,6 +425,13 @@ export class GlobalSearchInputComponent implements OnInit, OnDestroy {
     (<HTMLInputElement>document.activeElement).blur();
   }
 
+  public statusHighlightClass(statusId:any) {
+    if (!statusId) {
+      return;
+    }
+    return Highlighting.backgroundClass('status', statusId!);
```
Comment (ulferts) [inline frontend/src/app/modules/global_search/input/global-search-input.component.ts]: There is an a method in the `global-search-input.component` called `statusHighlighting`. It uses the `inlineClass` as opposed to the backgroundClass as done here.

Personally, I find the bubble in front of the status way more readable but it is probably personal preference. 

In any case, one of the methods should be removed.

**Review by ulferts — CHANGES_REQUESTED** [review]
Please see my remarks in the code.

File: frontend/src/app/modules/global_search/input/global-search-input.component.ts (hunk 1 above)
Comment (bsatarnejad) [reply frontend/src/app/modules/global_search/input/global-search-input.component.ts]: Hi @ulferts thanks for your feedback, I agree with you.

**Review by ulferts — APPROVED** [review]
👍

**Pull request merged**